Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1412,6 +1419,18 @@ def lhegensim2018(fragment,howMuch):
     global step1LHEGenSimUp2018Default
     return merge([{'cfg':fragment},howMuch,step1LHEGenSimUp2018Default])
 
+def lhegensim2018RD(fragment,howMuch):
+    global step1Up2018Defaults
+    return merge([{'cfg':fragment},howMuch,{'--customise_commands': "\"process.source.numberEventsInLuminosityBlock=cms.untracked.uint32(1000) \\n process.GlobalTag.toGet = cms.VPSet( cms.PSet(     record = cms.string('EcalLaserAPDPNRatiosRcd'),  tag = cms.string('EcalLaserAPDPNRatios_Run_Dep_MC'),  connect = cms.string('frontier://FrontierProd/CMS_CONDITIONS') ) ) \""},step1Up2018Defaults])
+#
+#* EcalLaserAPDPNRatios_Run_Dep_MC          (9IOVs)   run numbers: 1, 316082, 316720, 317527, ...
+#* EcalLaserAPDPNRatios_Run_Dep_MC_first_IOV   (1 IOV)   ---> used in SIM?
```

Comment:
if we go with multi-IOVs then we keep only EcalLaserAPDPNRatios_Run_Dep_MC correct?